src/components/Tool/NavToolList/Item/Item.tsx

```diff
@@ -22,29 +25,46 @@ interface Props {
   url: string;
   title: string;
   isSelected: boolean;
+  loading: boolean;
+  needsAttention: boolean;
+  totalCount: number;
```

Comment:
Do you think a name like `countNeedingAttention` would be more descriptive? I think it would be great if the prop name told what the count is of.